features/lifestage_page.feature

```diff
@@ -6,35 +6,33 @@ Feature: Lifestage page
   Scenario: Visiting an Lifestage page
     Given I entered into the Lifestage page "Young Adults"
     Then I should see the lifestage title "Young Adults"
-    And I should see the hero description "Research suggests that young adults typically display lower levels of financial capability than older age groups."
+    And I should see the hero description "Research suggests that Young Adults typically display lower levels of financial capability than other age groups."
     And I should see the lifestage content
     """
-    Young adults are those who have left compulsory education or other statutory settings,
-    such as care. They are transitioning towards independent living and financial independence,
-    beginning between the ages of 16 to 18 and continuing to their mid-20s.
+    This is an example paragraph containing information about a specific lifestage.
     """
     And I should see the latest news box with "1" news items
     And I should see the news items details
-      | title                            | date                  | link                                    |
-      | Press Release: A new way to pay! | Thursday 26 July 2018 | /en/news/press-release-a-new-way-to-pay |
+      | title  | date                     | link            |
+      | News 1 | Monday 19 November 2018  | /en/news/news-1 |
     And I should see the teaser boxes with
-      | title          | text                                                                                                              | link                    |
-      | Teaser 3 title | Research suggests that young adults typically display lower levels of financial capability than older age groups. | /exectutive+summary.pdf |
-      | Teaser 3 title | Research suggests that young adults typically display lower levels of financial capability than older age groups. | /exectutive+summary.pdf |
-      | Teaser 3 title | Research suggests that young adults typically display lower levels of financial capability than older age groups. | /exectutive+summary.pdf |
+      | title                       | text                                              | link                               |
+      | Some title to tease you     | Loads of content to make you read more            | /financial+capability+strategy.pdf |
+      | Another title to entice you | A bunch of well written content to make you click | /financial+capability+strategy.pdf |
+      | Teasing title               | You want to read this, you need to read this      | /financial+capability+strategy.pdf |
```

Comment:
Ow joy 😂. Is this is located on cms seeds?

Reply:
yes